src/index.less

```diff
@@ -161,6 +161,25 @@ ovh-chatbot { /* stylelint-disable-line selector-type-no-unknown */
       padding: 17px 17px 0 17px;
       height: ~'calc(100% - @{chatbot-header-height} - @{chatbot-footer-height})';
 
+      .chatbot-banner {
+        margin: -17px -17px 0 -17px;
+
+        .ovh-bandeau {
+          padding: 12px 17px;
+          text-align: center;
+
+          h3 {
+            font-size: 16px;
+          }
+        }
+
+        .ovh-incident {
+          color: #8a5300;
```

Comment:
Colors could be turned into variables in order to facilitate changes. Is that make sense to you?

Reply:
Yes :+1: